auction-deploy/auction_deploy/cli.py

```diff
@@ -141,3 +153,81 @@ def deploy(
     click.echo("Auction address: " + contracts.auction.address)
     click.echo("Deposit locker address: " + contracts.locker.address)
     click.echo("Validator slasher address: " + contracts.slasher.address)
+
+
+@main.command(
+    short_help="Prints the values of variables necessary to monitor the auction."
+)
+@click.option(
+    "--auction-address",
+    help='The address of the auction contract to be checked, "0x" prefixed string',
+    type=str,
+    required=True,
+)
+@jsonrpc_option
+def print_auction_status(auction_address, jsonrpc):
+
+    web3 = connect_to_json_rpc(jsonrpc)
+
+    contracts = get_deployed_auction_contracts(web3, auction_address)
+
+    # constants throughout auction
+    duration_in_days = contracts.auction.functions.auctionDurationInDays().call()
+    start_price_in_eth = contracts.auction.functions.startPrice().call() / ETH_IN_WEI
+    number_of_participants = contracts.auction.functions.numberOfParticipants().call()
+    locker_address = contracts.locker.address
+    slasher_address = contracts.slasher.address
+    locker_initialized = contracts.locker.functions.initialized().call()
+    slasher_initialized = contracts.slasher.functions.initialized().call()
+
+    # variables
+    auction_state_value = contracts.auction.functions.auctionState().call()
+    auction_state = AuctionState(auction_state_value)
+    start_time = contracts.auction.functions.startTime().call()
+    close_time = contracts.auction.functions.closeTime().call()
+    closing_price = contracts.auction.functions.closingPrice().call()
+    if auction_state == AuctionState.Started:
+        current_price_in_eth = (
+            contracts.auction.functions.currentPrice().call() / ETH_IN_WEI
+        )
+
+    click.echo(
+        "The auction duration is:                " + str(duration_in_days) + " days"
```

Comment:
I think in general we should better use string interpolation, so either `"this is a {}".format(variable)"` or `f"this is a {variable}"